gnomad_qc/v4/annotations/vrs_annotation_batch.py

```diff
@@ -401,6 +415,31 @@ def main(args):
         default=None,
         type=int,
     )
+    input_args = parser.add_mutually_exclusive_group(required=True)
+    input_args.add_argument(
+        "--data-type",
+        help="Data type to annotate (exomes or genomes).",
+        default="exomes",
```

Comment:
Since data-type is now part of a mutually exclusive group and needs to be specified to run, I don't think there needs to be a default anymore